layouts/_default/baseof.html

```diff
@@ -28,6 +28,7 @@ <h1 class="header__site-name">
 {{ if $.Param "menuinheader" }}{{ partial "menu.html" . }}{{ end -}}
 </div>
 {{ if $.Param "mobilemenu" }}{{ partial "mobilemenu.html" . }}{{ end -}}
+{{ if $.Param "math" }}{{ partial "math.js" . }}{{ end -}}
```

Comment:
Load the new katex.html, see below, here. Just replace "math.js" with "katex.html"

Or should the partial be named "math" instead of "katex"?

Pick what you think makes it most obvious for people interested in this. Then name the partial, the parameter etc. all the same so everyone can see they hang together.

katex: true -> katex.html -> …

or 

math: true -> math.html -> …